### PR TITLE
✨[Feature Request]: Creation of Blood Test

### LINKED ISSUE #874: ✨[Feature Request]: Blood Test
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Wanted to create a webpage for blood test.

### Add ScreenShots

image

Page is redirected when pointer on the above

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#874

## Description

[Please include a brief description of the changes or features added]

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #874 

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/5dc85f78-bf43-43a7-8e12-6f6e6cf52db2

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the changes.

**Comment by varshith257** [comment]
@Anjaliavv51 Unable to access the blood test from home page in deploy

**Comment by Anjaliavv51** [comment]
@varshith257 please click on the box which contains blood test hred

**Comment by varshith257** [comment]
In deploy it is not getting access for me

**Comment by Anjaliavv51** [comment]
@varshith257 but I am able to access it

File: index.html [hunk 1]
```diff
@@ -136,15 +136,18 @@ <h4 class="title"><a href=""> Normal checkup</a></h4>
                 cause problems.</p>
             </div>
           </div>
-
+          <a href="./Html-Files/bloodtest.html">
```
[the remark this answers is not in the document]
Comment (Anjaliavv51) [reply index.html:139]: Sure

**Comment by Anjaliavv51** [comment]
@varshith257 please review the changes.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the changes.

**Review by varshith257 — APPROVED** [review]

**Pull request merged**